tracks/python/exercises/pig-latin/pig-latin/mentoring.md

```diff
@@ -0,0 +1,42 @@
+# Pig Latin
+
+
+## Common Areas of Improvement
+When mentoring students on the Pig Latin exercise, it's essential to focus on common areas where they might struggle. Here are some key points to keep in mind:
+
+1. Understanding the Rules Clearly
+    - Rule Comprehension: Ensure that students fully understand each of the Pig Latin translation rules before diving into coding. Misunderstanding the rules often leads to incorrect implementations.
+    - Edge Cases: Highlight the importance of edge cases such as words starting with "xr" or "yt", and those containing "qu".
+2. String Manipulation Skills
+    - Splitting and Joining Strings: Students should be comfortable using functions to split sentences into words and join words back into sentences.
```

Comment:
If the solution passes tests, this is presumably not something that the mentor needs to bring up.